blog/_posts/2016-10-6-start-nonprofit.md

```diff
@@ -0,0 +1,246 @@
+that this was a great idea.
+
+It seems like DC could pretty easily switch over to using the URS and fully
+support the standard, or otherwise should not introduce an additional wrinkle
+into the registration process.
+
+Anyway, combining the various resources that we've found, these seem to be the main steps:
+
+## Organize information and other prep work
+
+### Draft mission statement
+
+This is not specifically a required step, but it is a useful exercise and will
+come in handy as you fill out forms again and again. The mission statement
+should be short and to the point. You can use the [OpenGov
+Foundation's](https://github.com/opengovfoundation/legal-information/blob/master/bylaws.md#section-1-purposes)
+as an example.
+
+### Draft Articles of Incorporation
+
+These will be submitted later, but you should probably do it now. They're
+relatively short and hit a few key points that DC's Superintended of
+Corporations cares about. You'll get to use that mission statement here. The
+OpenGov Foundation's certificate of incorporation (which you'll receive after
+filing the DNP-1 later) [includes their articles of
+incorporation](https://github.com/opengovfoundation/legal-information/blob/master/dc-certificate-of-incorporation.pdf)
+
+### Draft bylaws
+
+These will also come into play later, but are good to do now rather than rush
+once the organization is already up and running. The bylaws set forth the actual
+rules for operation of the organization, and thus set the tenor for the overall
+function. As before, you can use the [OpenGov Foundation's
+bylaws](https://github.com/opengovfoundation/legal-information/blob/master/bylaws.md)
+as a guide, but note that they have both a Board of Directors and a Board of
+Advisors, the latter of which may not be relevant to you.
+
+As part of this step, you'll also have to consider whether your organization
+will have "members". This has a specific legal meaning of somebody who votes in
+the election of directors and things like that. [D.C. Code §
+29-401.02(24)](https://beta.code.dccouncil.us/dc/council/code/sections/29-401.02.html).
+They are not required to form a nonprofit. [D.C. Code § 29-404.01(a)
+](https://beta.code.dccouncil.us/dc/council/code/sections/29-404.01.html). We
+decided that we probably do not need members, but other organizations may find
+them useful.
+
+### Plan out the Board of Directors
+
+This is listed as "establishing" a board of directors in a variety of places,
+but that seems difficult to do before the organization actually exists. Instead,
+we think this means that you should contact people and line them up to join the
+board as soon as it is possible. You'll need to have at least 3 of them. [D.C.
+Code §
+29-406.03(a)](https://beta.code.dccouncil.us/dc/council/code/sections/29-406.03.html).
+We couldn't tell immediately if officers can also be directors, but as a
+practical matter they shouldn't.
+
```

Comment:
I think so, too, but I'm still looking for something that actually says it one way or the other.